src/main/java/browserview/BrowserComponent.java

```diff
@@ -494,4 +496,15 @@ public void switchToTab(String tabName) {
             driver.findElement(By.xpath("//a[@data-container-id='" + tabName + "_bucket']")).click();
         }
     }
+
+    public Optional<String> getPRNumberFromIssue() {
+        try {
+            WebElement element = driver.findElement(By.xpath("//div[contains(@id, 'ref-pullrequest')]"));
+            return Optional.of(element.findElement(By.xpath("..")).findElement(By.xpath(".//h3/a/span"))
+                    .getAttribute("innerHTML").substring(1));
```

Comment:
Will this get the first PR if an issue mentions a few? Maybe we can add a comment saying what it does.